templates/master/00-master/gcp/files/etc-kubernetes-manifests-apiserver-watcher.yaml

```diff
@@ -0,0 +1,38 @@
+mode: 0644
+path: "/etc/kubernetes/manifests/apiserver-watcher.yaml"
```

Comment:
Hmm...does really need to be a static pod versus part of the existing MCD?  I guess since the previous code was a static pod too it's OK to keep as is for now.

Reply:
Yes, that's probably the only reason.

Reply:
I wrote some small code to teach the MCD to behave differently on the control plane as part of https://github.com/openshift/machine-config-operator/pull/1946 FWIW.